recipe/meta.yaml

```
@@ -1,52 +1,59 @@
 {% set name = "celery" %}
-{% set version = "5.0.5" %}
+{% set version = "5.5.3" %}
+{% set u_test_files_to_skip = "--ignore=t/unit/app/test_preload_cli.py " %}
+{% set u_test_files_to_skip = u_test_files_to_skip + "--ignore=t/unit/bin/test_beat.py " %}
+{% set u_test_files_to_skip = u_test_files_to_skip + "--ignore=t/unit/bin/test_control.py " %}
+{% set u_test_files_to_skip = u_test_files_to_skip + "--ignore=t/unit/bin/test_worker.py " %}
+{% set u_test_files_to_skip = u_test_files_to_skip + "--ignore=t/unit/bin/test_daemonization.py " %}
+{% set u_tests_to_skip = " test_check_privileges " %}
+{% set u_tests_to_skip = u_tests_to_skip + "or test_pytest_celery_marker_registration " %}
+{% set u_tests_to_skip = u_tests_to_skip + "or test_on_event_task_received " %}
+{% set u_tests_to_skip = u_tests_to_skip + "or test_on_event_non_task " %}
 
 package:
-    name: {{ name|lower }}
-    version: {{ version }}
+  name: {{ name|lower }}
+  version: {{ version }}
 
 source:
-    url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-    sha256: f4efebe6f8629b0da2b8e529424de376494f5b7a743c321c8a2ddc2b1414921c
-    patches:
-        - 0001-skip-a-test.patch
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6c972ae7968c2b5281227f01c3a3f984037d21c5129d07bf3550cc2afc6b10a5
 
 build:
-    noarch: python
-    number: 0
-    script: "{{ PYTHON }} -m pip install . --no-deps -vv"
-    entry_points:
-        - celery = celery.__main__:main
+  skip: true  # [py<38]
+  entry_points:
+    - celery = celery.__main__:main
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
 
 requirements:
-    build:
+    host:
         - python
         - pip
```

https://github.com/celery/celery/blob/v5.5.3/setup.py#L6
```suggestion
        - pip
        - wheel
        - setuptools
```